docs/cli.md

```diff
@@ -102,11 +102,17 @@ my-package
 
 ### Options
 
+* `--interactive (-i)`: Allow interactive specification of project configuration.
```

Comment:
Why do we need this over `io.is_interactive()`?

Reply:
For `poetry new` non-interactive is the current expectation, even if `--no-interaction` is not explicitly set. So we need a way to trigger the "interactive" configuration when someone needs it to bring parity with `poetry init`.